psydac/core/bsplines.py

```diff
@@ -600,5 +600,21 @@ def basis_ders_on_quad_grid( knots, degree, quad_grid, nders ):
             ders = basis_funs_all_ders( knots, degree, xq, span, nders )
             basis[ie,:,:,iq] = ders.transpose()
 
+    if normalize:
```

Comment:
What kind of normalization is this? If we are requiring the basis functions to integrate to 1 instead of having the partition of unity property, then we should explain this carefully in the docstring of this function.